src/PrestaShopBundle/Resources/config/services/adapter/category.yml

```diff
@@ -0,0 +1,50 @@
+services:
+    _defaults:
+        public: true
+
+    prestashop.adapter.category.command_handler.abstract_category_handler:
+        class: 'PrestaShop\PrestaShop\Adapter\Category\CommandHandler\AbstractCategoryHandler'
+        abstract: true
+        arguments:
+            - '@prestashop.adapter.image.uploader.category_cover_image_uploader'
+            - '@prestashop.adapter.image.uploader.category_thumbnail_image_uploader'
+            - '@prestashop.adapter.image.uploader.category_menu_thumbnail_image_uploader'
+
+    prestashop.adapter.category.command_handler.add_category_handler:
+        class: 'PrestaShop\PrestaShop\Adapter\Category\CommandHandler\AddCategoryHandler'
+        parent: 'prestashop.adapter.category.command_handler.abstract_category_handler'
+        public: true
+        tags:
+            - { name: 'tactician.handler', command: 'PrestaShop\PrestaShop\Core\Domain\Category\Command\AddCategoryCommand' }
```

Comment:
be consistent in your writing, either write on one line or two lines But use the same syntax in the whole file